src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -2425,6 +2425,13 @@ Rate.InstanceResetTime = 1
 #
 Rate.Pet.LevelXP = 0.05
 
+#
+#    WaterBreath.Timer
+#        Description: The timer for player's breath underwater in minutes
+#        Default:     3
+#
```

Comment:
enter

Reply:
Fixed, thanks. I also added a missing enter to the previous entry, which also was missing.